protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -25,6 +25,12 @@ import (
 )
 
 var wktSchemas = map[string]schemaCore{
+	".google.protobuf.FieldMask": {
+		Type: "array",
+		Items: (*openapiItemsObject)(&schemaCore{
+			Type: "string",
+		}),
+	},
```

Comment:
Could you add a comment here stating that this must match up with the behaviour of our JSON unmarshaler, e.g. https://github.com/protocolbuffers/protobuf-go/blob/v1.25.0/encoding/protojson/well_known_types.go#L860-L885. It looks to me like it interprets a comma-separated slice of paths, so this description looks right.

Reply:
I added a comment